package.json

```diff
@@ -191,6 +191,11 @@
         "title": "Restart Ganache",
         "category": "Truffle"
       },
+      {
+        "command": "truffle-vscode.getGanacheServerInfo",
+        "title": "Get Details...",
+        "category": "Truffle"
+      },
```

Comment:
We might need to either add this command to `activationEvents`, or remove it from the Command Palette altogether. Otherwise this error is thrown when I try to run the command before the extension has been activated.

<img width="259" alt="image" src="https://user-images.githubusercontent.com/4592980/188908136-0d81d631-ffd7-4abf-83d8-1f5c31dff022.png">

Moreover, if I run the command from the Command Palette after the extension has been activated, this empty tab appears, regardless of if any Ganache service is running.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/4592980/188909031-9ec70804-5b27-40b6-bbab-6b71212a2531.png">

Reply:
You're right @acuarica. I'm gonna vanish it out from the Command Palette

Thanks!